package.json

```diff
@@ -27,10 +27,10 @@
     "prebuild": "npm run clean",
     "prepublish": "npm run build",
     "release": "standard-version",
-    "security": "nsp check",
+    "security": "npm audit --json --audit-level high | npm-audit-helper --prod-only",
```

Comment:
No need `npm-audit-helper`, just use `npm audit`

Reply:
It will fail then because `npm audit` fails because devDependencies have vulnerabilities, and one of them, `babel` cannot be upgraded without dropping support for Node 4 and 6

Reply:
drop support node@4

Reply:
Had to drop support for node 4 & 6